src/back-end/lib/routers/auth.ts

```diff
@@ -135,8 +135,58 @@ async function makeRouter(connection: Connection): Promise<Router<any, any, any,
           return makeAuthErrorRedirect(request);
         }
       })
-    }
+    },
+    // /auth/createsession is for testing only, so only exists outside of production
+    // @ts-ignore
```

Comment:
Since this is a testing endpoint, not a real one, it doesn't return the types the app expects. OK to use @ts-ignore here and below?

Reply:
Ideally we'd cast this endpoint's output to return prod types. Tech debt

Reply:
https://github.com/button-inc/digital_marketplace/issues/119